test/distributed/tensor/parallel/test_micro_pipeline_tp.py

```diff
@@ -82,14 +84,12 @@ def tearDown(self):
     def test_find_all_gather_patterns(self):
         group = dist.group.WORLD
 
-        def func(inp: torch.Tensor) -> torch.Tensor:
+        def func(inp: torch.Tensor):
```

Comment:
Why'd you remove the return type here, and why doesn't it complain? 🙂

Reply:
The type is wrong, it should be tuple of Tensors.